src/config.js

```diff
@@ -578,7 +558,7 @@ export function newConfig() {
       .forEach(listener => listener.callback(options));
   }
 
-  function setBidderConfig(config) {
+  function setBidderConfig(config, flag = false) {
```

Comment:
I suggest using a more descriptive name for flag. I think 'deep' or 'deepMerge' would describe the intent.

Reply:
Agreed

Reply:
I landed on 'mergeFlag'. 'deepMerge' seems a bit confusing since 'mergeDeep' is on the same line. Let me know your thoughts. If you don't like 'mergeFlag' then I think 'merge' makes more sense than 'deep'.

Reply:
I think mergeFlag is fine for this. One thing to note is that along with adding mergeConfig/mergeBidderConfig to the docs, we may want to make updates to setBidderConfig as it now includes two arguments